BraintreeCore/src/main/java/com/braintreepayments/api/ShopperInsightsRequest.kt

```diff
@@ -0,0 +1,11 @@
+package com.braintreepayments.api
+
+/**
+ * Data class representing a request for shopper insights.
+ *
+ * Note: **This feature is in beta. It's public API may change in future releases.**
+ */
+data class ShopperInsightsRequest(
+    var email: String?,
```

Comment:
Can we add docs for these properties? 

Ex:
email The buyer's email address. If not set, then [phone] is required.